Magento.gitignore

```diff
@@ -45,8 +45,43 @@ app/etc/local.xml
 app/.htaccess
 app/locale/
 app/Mage.php
+app/bootstrap.php
 /cron.php
 cron.sh
+/dev/tests/functional/etc
+/dev/tests/functional/lib/Magento/Mtf
+/dev/tests/functional/tests/app/Mage
+/dev/tests/functional/testsuites/Magento/Mtf/TestSuite/InjectableTests/3rdParty.xml
```

Comment:
I will take a look at it. I just installed it out of the box and made the assumption that they are there to extend. But I will validate it :+1: